script/dev/dev-UpgradeLockingContract.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
We shouldn't copy over these contracts and should import them from mento-core instead. e.g.: https://github.com/mento-protocol/mento-deployment/blob/0efbfe2df2844e71570608cb0478a771f657c881/script/upgrades/MU08/deploy/MU08-00-Create-Proxies.sol#L8

We can create a new release in mento-core and tag it once the audit changes are merged into develop